Source/relay/TourGuide/Support/IOTMs.ash

```diff
@@ -25,10 +25,14 @@ void initialiseIOTMsUsable()
         if (__campground[lookupItem("cold medicine cabinet")] > 0)
             __iotms_usable[lookupItem("cold medicine cabinet")] = true;
 
-        // Garden
-        if (__campground[lookupItem("packet of mushroom spores")] > 0)
-            __iotms_usable[lookupItem("packet of mushroom spores")] = true;
-
+        // __iotms_usable for gardens tracks whether the user has the garden installed.
+        // Gardens start returning 0 instead of 1 when the items are picked, so checking
+        // presence of the key is more useful than checking the value.
+        foreach garden in $items[packet of mushroom spores, packet of rock seeds]
```

Comment:
glad we were able to get to the bottom of this one in chat. logic looks right here.